docs/source/apidoc.md

```diff
@@ -365,6 +365,12 @@ See Ref. {cite}`Czarnik_2021_Quantum` for more details on these methods.
 .. automodule:: mitiq.interface.mitiq_pyquil.conversions
    :members:
 ```
+#### Qibo Conversions
+
+```{eval-rst}
+.. automodule:: mitiq.interface.mitiq_qibo.conversions
+   :members:
+```
```

Comment:
Question: There are a bunch of functions in this module that _maybe_ don't need to go into the API-doc. E.g. all the `..._decomp` functions. Should we just have the `to_qibo` and `from_qibo` functions?